src/sbin/livemedia-creator

```diff
@@ -663,20 +664,12 @@ def novirt_install(opts, disk_img, disk_size):
     :param str disk_img: The full path to the disk image to be created
     :param int disk_size: The size of the disk_img in MiB
 
-    This method makes sure SELinux is permissive during the install, runs anaconda
-    to create the image and then based on the opts passed create a qemu disk image
-    or tarfile.
+    This method runs anaconda to create the image and then based on the opts
+    passed create a qemu disk image or tarfile.
```

Comment:
"creates".  Pre-dates this change but might as well fix it as long as you're in there.